Common/include/config_structure.hpp

```diff
@@ -177,8 +180,11 @@ class CConfig {
   nMarker_SymWall,				/*!< \brief Number of symmetry wall markers. */
   nMarker_Pressure,				/*!< \brief Number of pressure wall markers. */
   nMarker_PerBound,				/*!< \brief Number of periodic boundary markers. */
-  nMarker_MixBound,				/*!< \brief Number of mixing boundary markers. */
-  nMarker_TurboPerf,				/*!< \brief Number of mixing boundary markers. */
+  nMarker_MixingPlaneInterface,				/*!< \brief Number of mixing plane interface boundary markers. */
+  nMarker_Turbomachinery,				/*!< \brief Number turbomachinery markers. */
+  nMarker_TurboPerformance,				/*!< \brief Number of turboperformance markers. */
+  nSpanWiseSections_User,			/*!< \brief Number of spanwise sections to compute 3D BC and Performance for turbomachinery   */
+  nMarker_Shroud,/*!< \brief Number of shroud markers to set grid velocity to 0.*/
```

Comment:
Sounds great! 
So is this, basically, a marker to specify a specific value of fluid speed over a certain boundary (which is exactly the speed of the grid)?
Cause I think that such possibility could be very useful also in applications other than turbomachinery.
So, maybe in the future, we could think to create a general BC to allow the user to specify the velocity over the boundary.